components/fastly_frontend/main.tf

```diff
@@ -159,7 +169,7 @@ resource "fastly_service_v1" "frontend" {
     name    = "frontend"
     address = element(split(":", var.papertrail_destination), 0)
     port    = element(split(":", var.papertrail_destination), 1)
-    format  = var.papertrail_log_format
+    format  = "%t '%r' status=%>s app=%%{X-Application-Name}o cache=\"%%{X-Cache}o\" country=%%{X-Fastly-Country-Code}o ip=\"%a\" user-agent=\"%%{User-Agent}i\" service=%%{time.elapsed.msec}Vms"
```

Comment:
Like in #270, I've inlined this here since we always want the same log format regardless of what environment we're in. (We may, however, eventually want different log formats between frontend & backend properties).